### PR TITLE
further yogsmeta improvements

### PR DESCRIPTION
**Pull request by athena413**
# General Documentation

### Intent of your Pull Request

Improves Meta a bit more.

All changes listed on: https://pastebin.com/DBLaetRG

Highlights include:

CMO now spawns in the meeting room

Bridge meeting room is now larger - main bridge area is a bit smaller now.

Paintings have been added.

Clerk office flipped to face south

### Why is this change good for the game?

better map is good

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

This PR makes the following articles majorly outdated:
https://wiki.yogstation.net/wiki/Bridge
https://wiki.yogstation.net/wiki/Conference_Room

Most images of Yogsmeta are now mildly outdated.

### What should players be aware of when it comes to the changes your PR is implementing?

See above.

### What general grouping does this PR fall under? 

Mapping

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

N/A

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

N/A

# Changelog

:cl:  
tweak: the entire map
/:cl:

### CODE REVIEW

**Comment by athena413** [comment]
fucker posted even though i didnt press enter. hold on

**Review by patpol4 — CHANGES_REQUESTED** [review]
>Meeting area is now bigger

Why, it's just a room heads start in, the bridge is more important. Making it bigger doesnt make much sense to me

>Clerk rotation

Undocumented, also this part can't stay like this
image

>Many many paintings

Way way too many , can you reduce them by about a quarter because there sems to be one around every single corner of the map, Station isn't an art gallery, just keep it to a similar amount as box.

**Comment by athena413** [comment]
> Why, it's just a room heads start in, the bridge is more important. Making it bigger doesnt make much sense to me

there wasnt any other way to put the CMO in there without making the spawns be kinda shit

> Undocumented, also this part can't stay like this

swear to god i documented that, will fix

> Way way too many , can you reduce them by about a quarter because there seems to be one around every single corner of the map, Station isn't an art gallery, just keep it to a similar amount as box.

will do

**Review by patpol4 — CHANGES_REQUESTED** [review]
I still dislike the bridge change, and the middle table can't be reached by normal means.
vivaldi_G2lCQte9Ep

Something like this could work 
image

**Comment by athena413** [comment]
im LOAing for a month

will redo this if i feel like it when im done
